src/Types/UpdatePayload.cs

```diff
@@ -16,28 +16,51 @@ public interface IUpdatePayload
 /// </summary>
 public enum UpdateType
 {
+    /// <summary> Update of protocol version. </summary>
```

Comment:
I love doc comments more than the average person probably but a lot of these doc comments really add nothing. Is there nothing more to add? Or could we not disable whatever requirement is forcing these i.e. akin to Rust's `#[allow(missing_docs)]`?

Reply:
I don't think there is a lot more to add to this.
To me, this still seems better than disabling the warning.